packages/reusable-blocks/src/components/reusable-blocks-menu-items/index.js

```diff
@@ -10,7 +10,11 @@ import { store as blockEditorStore } from '@wordpress/block-editor';
 import ReusableBlockConvertButton from './reusable-block-convert-button';
 import ReusableBlocksManageButton from './reusable-blocks-manage-button';
 
-function ReusableBlocksMenuItems( { clientIds, rootClientId } ) {
+export default function ReusableBlocksMenuItems( { rootClientId } ) {
+	const clientIds = useSelect( ( select ) =>
+		select( blockEditorStore ).getSelectedBlockClientIds()
+	);
```

Comment:
Should pass `[]` as the `deps` for `useSelect` here. Annoyingly it doesn't default to `[]`.

Reply:
thanks, fixed, we should have a lint rule for that.